gradle.properties

```diff
@@ -1,7 +1,6 @@
 org.gradle.jvmargs='-Dfile.encoding=UTF-8'
 
 GROUP=com.squareup
-VERSION_NAME=1.14.0-SNAPSHOT
```

Comment:
Doesn't this break the ability to do Maven local deploys for testing?

Reply:
It does, yep. This likely breaks more things, and will generally be non-ergonomic for local development.